rust/release-checklist.yaml

```diff
@@ -2,18 +2,21 @@ vars:
   do_fast_track: true
   do_release_digests: true
   do_vendor_tarball: true
+  do_vendor_filter: true
 
 files:
   - repo: afterburn
     path: .github/ISSUE_TEMPLATE/release-checklist.md
     vars:
       do_release_notes_doc: true
 
+  - repo: bootupd
+    path: .github/ISSUE_TEMPLATE/release-checklist.md
```

Comment:
This will clobber the existing release checklist in that repo.  I think the main consequence is that we'll need to remove `ci/prepare-release.sh`, but there might be other things as well.  Perhaps we should confirm this change with a repo maintainer before landing it here.